### PR TITLE
EZP-23866: Move legacy_mode setting to LegacyBundle

### PR DESCRIPTION
**Pull request by lolautruche**
> https://jira.ez.no/browse/EZP-23866

See https://github.com/ezsystems/ezpublish-kernel/pull/1135

### CODE REVIEW

**Comment by yannickroger** [comment]
+1

File: ezpublish/config/config.yml [hunk 1]
```diff
@@ -50,10 +50,6 @@ assetic:
         #yui_css:
         #    jar: "%kernel.root_dir%/Resources/java/yuicompressor-2.4.7.jar"
 
-ez_publish_legacy:
-    enabled: true
-    root_dir: %kernel.root_dir%/../ezpublish_legacy
```
Comment (andrerom) [inline ezpublish/config/config.yml]: not needed anymore?

**Comment by andrerom** [comment]
besides question, +1

File: ezpublish/config/config.yml (hunk 1 above)
Comment (lolautruche) [inline ezpublish/config/config.yml]: Nope. `enabled` is now true by default (needs LegacyBundle to be instantiated, and `root_dir` has `%kernel.root_dir%/../ezpublish_legacy` as default value.

**Comment by lolautruche** [comment]
> is the BDD failures fixed with the PR in kernel?

It should yes. But I can update `composer.json` so we can check it out.

**Comment by lolautruche** [comment]
There, all green :smile:

**Pull request merged**